### DISCUSSION TITLE
SpeedBoost

### DISCUSSION THREAD

**Discussion by Nyaster** [vladmandic/sdnext#181] [Ideas]
Any ideas about implementing VoltaML or Aitemplate(I know, this required model conversion), but after conversion you have x2 speed(on my laptop 3070(128w power limit)) i have 19it/s on 512x512 vs around 8it/s right now.

**Comment by vladmandic** [comment]
Ideas, yes, but not that trivial to properly integrate workflows, going to be a while.

**Reply by Nyaster** [reply]
Mostly this be profit if using with hires fix, with tomesd its very big speedboost.
